graylog2-server/src/main/java/org/graylog/plugins/views/search/SearchType.java

```diff
@@ -121,6 +127,10 @@ class Fallback implements SearchType {
         @JsonProperty
         private Filter filter;
 
+        @Nullable
```

Comment:
Could we avoid a nullable field and make sure that the list is at least empty? Or would it change the meaning of the logic?

Reply:
It is present in Fallback implementation only. It tends to use this annotation on many other fields, so I'd keep it as it is, for consistency.